realtime/src/config/yaml.ts

```diff
@@ -54,3 +54,17 @@ export const getExchangesConfig = (): ExchangeConfig[] =>
         config: e.config,
       }
     })
+
+  const exchangesConfig: ExchangeConfig[] = []
+  for (const config of rawExchangesConfig) {
+    let newConfigs = [config]
+    if (config.quote === "*" || config.quoteAlias === "*") {
+      newConfigs = supportedCurrencies
+        .filter((e) => e.code !== config.base)
+        .map((c) => ({ ...config, quoteAlias: c.code, quote: c.code }))
```

Comment:
Is there ever a case where the `quote` isn't the same as the currency `code` (e.g. if `quote` value is `USDT`)?

Reply:
yep, with some exchanges https://github.com/GaloyMoney/price/blob/8e3c3b56be28f5ba56fa053826c90b2eac6d04f8/realtime/default.yaml#L25 but for wildcard should be the same, if we want a different quote/quote alias should be a different config

Reply:
Ok makes sense. Approving, but we should probably document somewhere that supported currency `base` should be compared against exchange `code` before configuring new wildcards or supported currencies.